src/hashtable.h

```diff
@@ -96,6 +96,7 @@ typedef void (*hashtableScanFunction)(void *privdata, void *entry);
 /* Iterator flags */
 #define HASHTABLE_ITER_SAFE (1 << 0)
 #define HASHTABLE_ITER_PREFETCH_VALUES (1 << 1)
+#define HASHTABLE_ITER_INCLUDE_IMPORTING (1 << 1)
```

Comment:
Duplicate number. Need to bump this.

```suggestion
#define HASHTABLE_ITER_INCLUDE_IMPORTING (1 << 2)
```